src/functions.cpp

```diff
@@ -24,3 +24,6 @@ int min(std::vector<int> const & values) {
     return minimum;
 }
 
+void reverse(std::vector<int> & data) {
+    std::reverse(begin(data), end(data));
+}
```

Comment:
Newlines are missing

Reply:
but why there are needed? Is there a purpose in the new empty line?

Reply:
@nauka-programowania-MA wrote somewhere: "According to the ANSI standard, when you #include another file, the compiler may insert the contents without including a newline at the end of the #include (though realistically, any modern compiler will put a newline at the end). An example of where this might cause problems: if your header file "foo.h" ends in a comment (instead of a newline), then your include could result in commenting out the first line of "foo.cpp"."